fbw-a380x/src/wasm/systems/a380_systems/src/electrical/mod.rs

```diff
@@ -58,9 +58,11 @@ impl A380Electrical {
     const MIN_EMERGENCY_GENERATOR_RPM_TO_ALLOW_CURRENT_SUPPLY: f64 = 2000.;
 
     const RAT_CONTROL_SOLENOID1_POWER_BUS: ElectricalBusType =
-        ElectricalBusType::DirectCurrentHot(1);
+        ElectricalBusType::DirectCurrentEssential;
+
+    //TODO second solenoid should be HOT BUS ESSENTIAL
     const RAT_CONTROL_SOLENOID2_POWER_BUS: ElectricalBusType =
-        ElectricalBusType::DirectCurrentHot(2);
+        ElectricalBusType::DirectCurrentEssential;
```

Comment:
```suggestion
        ElectricalBusType::DirectCurrentHot(3);
```
This is the correct bus. It will be available as soon as I finish the elec system